echo-scheduler/quartz_init.sql

```diff
@@ -0,0 +1,182 @@
+DROP TABLE IF EXISTS QRTZ_FIRED_TRIGGERS;
```

Comment:
The reason we should make the tables manually is because if we opt to use `QuartzAutoConfigure` - it will blow away and recreate the tables on start which is not what we want to support multiple instances. Perhaps I am missing something and we can configure it more better to only make tables if none are found?

Reply:
Ha. That doesn’t sound right. If we’re going to maintain a schema within the service, we should probably use liquibase to apply it so the operator experience is similar for other SQL services. Liquibase fortunately supports SQL migration files as well, so this shouldn’t be too much of a lift.

Reply:
makes sense, will do that

Reply:
Fixed